utils/cliutils/commandsflags.go

```diff
@@ -761,6 +762,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  explode,
 		Usage: "[Default: false] Set to true to extract an archive after it is downloaded from Artifactory.` `",
 	},
+	bypassArchiveInspection: cli.BoolFlag{
+		Name:  bypassArchiveInspection,
+		Usage: "[Default: false] Set to true to bypass archive inspections before exploding. Used with the 'explode' option. ` `",
```

Comment:
...to bypass archive inspections before exploding... --> ...to bypass the archive security inspection before it is unarchived...